[2024-09-28T11:55:36.096+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-28T11:55:36.105+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag_with_python_v4.get_name manual__2024-09-28T11:55:35.482622+00:00 [queued]>
[2024-09-28T11:55:36.108+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag_with_python_v4.get_name manual__2024-09-28T11:55:35.482622+00:00 [queued]>
[2024-09-28T11:55:36.109+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-28T11:55:36.115+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): get_name> on 2024-09-28 11:55:35.482622+00:00
[2024-09-28T11:55:36.121+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2317) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-28T11:55:36.121+0000] {standard_task_runner.py:72} INFO - Started process 2318 to run task
[2024-09-28T11:55:36.120+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_dag_with_python_v4', 'get_name', 'manual__2024-09-28T11:55:35.482622+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmpbeqk54hu']
[2024-09-28T11:55:36.122+0000] {standard_task_runner.py:105} INFO - Job 61: Subtask get_name
[2024-09-28T11:55:36.147+0000] {task_command.py:467} INFO - Running <TaskInstance: first_dag_with_python_v4.get_name manual__2024-09-28T11:55:35.482622+00:00 [running]> on host f9a59cccd6d1
[2024-09-28T11:55:36.183+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='first_dag_with_python_v4' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-09-28T11:55:35.482622+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-28T11:55:35.482622+00:00'
[2024-09-28T11:55:36.184+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-28T11:55:36.192+0000] {python.py:240} INFO - Done. Returned value was: Jerry
[2024-09-28T11:55:36.207+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-28T11:55:36.208+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_dag_with_python_v4, task_id=get_name, run_id=manual__2024-09-28T11:55:35.482622+00:00, execution_date=20240928T115535, start_date=20240928T115536, end_date=20240928T115536
[2024-09-28T11:55:36.222+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-28T11:55:36.233+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-28T11:55:36.234+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
